docs/reference/manifest/override.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: Override element in the manifest file
 description: The Override element enables you to specify the value of a setting depending on a specified condition.
-ms.date: 11/06/2020
+ms.date: 05/14/2021
 localization_priority: Normal
 ---
 
 # Override element
 
-Provides a way to override the value of a manifest setting depending on a specified condition. There are two kinds of conditions:
+Provides a way to override the value of a manifest setting depending on a specified condition. There are three kinds of conditions:
 
 - An Office locale that is different from the default.
 - A pattern of requirement set support that is different from the default pattern.
+- The runtime source is different from the default (currently in preview).
 
-There are two types of `<Override>` elements, one is for locale overrides, called **LocaleTokenOverride**, and the other for requirement set overrides, called **RequirementTokenOverride**. But there is no `type` parameter for the `<Override>` element. The difference is determined by the parent element and the parent element's type. An `<Override>` element that is inside of a `<Token>` element whose `xsi:type` is `RequirementToken`, must be of type **RequirementTokenOverride**. An `<Override>` element inside any other parent element, or inside an `<Override>` element of type `LocaleToken`, must be of type **LocaleTokenOverride**. Each type is described in separate sections below. For more information about the use of this element when it is a child of a `<Token>` element, see [Work with extended overrides of the manifest](../../develop/extended-overrides.md).
+There are three types of `<Override>` elements, one is for locale overrides, called **LocaleTokenOverride**, another for requirement set overrides, called **RequirementTokenOverride**, and a third for runtime overrides, called **RuntimeOverride**. But there is no `type` parameter for the first two types of `<Override>` element. The difference is determined by the parent element and the parent element's type. An `<Override>` element that is inside of a `<Token>` element whose `xsi:type` is `RequirementToken`, must be of type **RequirementTokenOverride**. An `<Override>` element that is inside of a `<Runtime>` element must be of type **RuntimeOverride**. An `<Override>` element inside any other parent element, or inside an `<Override>` element of type `LocaleToken`, must be of type **LocaleTokenOverride**. Each type is described in separate sections later in this article. For more information about the use of this element when it is a child of a `<Token>` element, see [Work with extended overrides of the manifest](../../develop/extended-overrides.md).
```

Comment:
So, now there IS a `type` attribute, but it is used to distinguish runtime types, not override types, so we need to reword this. 

```suggestion
There are three types of `<Override>` elements, one is for locale overrides, called **LocaleTokenOverride**, another for requirement set overrides, called **RequirementTokenOverride**, and a third for runtime overrides, called **RuntimeOverride**. But there is no `overrideType` attribute for the `<Override>` element. The difference is determined by the parent element and the parent element's type. An `<Override>` element that is inside of a `<Token>` element whose `xsi:type` is `RequirementToken`, must be of type **RequirementTokenOverride**. An `<Override>` element that is inside of a `<Runtime>` element must be of type **RuntimeOverride**. An `<Override>` element inside any other parent element, or inside an `<Override>` element of type `LocaleToken`, must be of type **LocaleTokenOverride**. Each type is described in separate sections later in this article. For more information about the use of this element when it is a child of a `<Token>` element, see [Work with extended overrides of the manifest](../../develop/extended-overrides.md).
```